general/server/users/adding-managing-users.md

```diff
@@ -12,32 +12,36 @@ User management can be done under `Users` in the `Dashboard`. Here you can see y
 
 ### Profile
-Directly under the tabs you have a link to `Edit this user's profile, image and personal preferences.` Clicking that allows you to change the user's personal settings, any setting here can be changed by both user and admin. <!--These options are described [on their own docs page](https://linktopage?) (not written yet)-->
+Directly under the tabs you have a link to `Edit this user's profile, image and personal preferences.` Clicking that allows you to change the user's personal settings, any setting here can be changed by both user and admin.
 
 Under `name` you can change the user's name as it will be either displayed on, or has to be typed into login screen.
 
 Under `Authentication Provider` you have the option to change the backend that handles the login, by default the only option here will be `default` which means Jellyfin will handle this user, this option is sufficient for most use cases. Currently, the only other possibility is to have a LDAP server handle the login by installing the LDAP-Auth plugin. Note that if you wish to change a user's provider to LDAP after creating it in Jellyfin the username needs to be identical to the user's UID in LDAP, including capitalization.
 
-`Allow remote connections to this Jellyfin Server.` Unchecking this option will block login attempts this user makes from outside the networks defined as local, by default this will only be the subnet assigned to your network. But more can be added. <!-- More information about this can be found [on the docs page for `Expert` » `Advanced`](https://linktopage?) (not written yet)-->
+`Allow remote connections to this Jellyfin Server.` Unchecking this option will block login attempts this user makes from outside the networks defined as local, by default this will only be the subnet assigned to your network. But more can be added.
 
 #### Feature Access
+
 For the following options it should be noted that if you never set up Live TV, users are blocked regardless of the state. See [the docs page for `Live TV` » `Live TV`](xref:server-live-tv-index) for more information.
 
 `Allow Live TV access` Unchecking this option will block the user's access to watch Live TV.
 
 `Allow Live TV recording management` Unchecking this option will block the user's access to set recording schedules.
 
 #### Media Playback
+
 `Allow media playback` Unchecking this option will block the user's access to media libraries, this does not include Live TV.
 
 > [!NOTE]

```

Comment:
We should change all those `**Note:**` notes to this syntax as this renders like a nicely colored panel.